### DISCUSSION TITLE
Question about Resignation

### DISCUSSION THREAD

**Discussion by wufanyou** [PaddlePaddle/PaddleSpatial#47] [BaiduKDDCup2022] [answered]
Is there a way to make a resignation for those people who do have a Chinese phone number right now? I tried to add 001 or + 1 but none of them works.

**Comment by zhoujingbo** [comment, marked as answer]
Hi, 

you can try +1XXXX (XXX is your phone number) if you are in the USA. 
Previously, we have successfully tested our systems to register in the US, Singapore, and Africa.

Regards
